client.go

```diff
@@ -102,6 +102,21 @@ type (
 		// The activity can fail with below errors ErrorWithDetails, TimeoutError, CanceledError.
 		CompleteActivity(ctx context.Context, taskToken []byte, result interface{}, err error) error
 
+		// CompleteActivityById reports activity completed.
+		// Similar to CompleteActivity, but may save cadence user from keeping taskToken info
```

Comment:
Fix the comment as it looks like multiple sentences without dots. The generated documentation removes all new lines (unless double new line).